### PR TITLE
Implemented Note list page UI

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by nitinprojects**
fixes #1

### CODE REVIEW

**Comment by nitinprojects** [comment]
I have fixed a few things in other PR

#15 - naming convention
#17 - use of dayjs

File: app/javascript/src/components/Dashboard/Notes/NoteList.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "@bigbinary/neeto-icons";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NoteList.jsx:3]: Please go through the Slack thread very carefully. I had already added a review comment for the same.

File: app/javascript/src/components/Dashboard/Notes/NoteList.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const NotesList = ({ data }) => (
+  <div className="flex w-full flex-col space-y-4">
+    {data.map(note => (
+      <div className="rounded-lg border bg-white p-4 shadow-sm" key={note.id}>
+        <div className="flex justify-between">
+          <Typography component="h2" lineHeight="relaxed" style="h4">
+            {note.title}
+          </Typography>
+          <Dropdown buttonStyle="text" icon={MenuVertical} label="">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NoteList.jsx:14]: Why do we have to pass an empty label?

File: app/javascript/src/components/Dashboard/Notes/NotesMenuBar.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { Settings, Plus, Search } from "@bigbinary/neeto-icons";
+import { MenuBar } from "@bigbinary/neetoui/layouts";
+import { Typography } from "neetoui";
+
+const NotesMenuBar = ({ showMenu }) => (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NotesMenuBar.jsx:7]: Why do we have to prefix with the `Notes` keyword for this component? What advantage does it provide?

Please read: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+    tag: "Onboarding",
+    status: "drafted",
+    createdAt: "2 hours ago",
+    createdBy: "Oliver Smith",
+    imageUrl: "https://i.pravatar.cc/300",
+  },
+  {
+    id: 3,
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:27]: Why is this a template string even when you aren't performing any interpolation?

**Review by yedhink — COMMENTED** [review]
@nitinprojects

**Pull request merged**